app/views/content/how-to-apply-for-teacher-training/teacher-training-application.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Your teacher training application
+title: Teacher training application
```

Comment:
Do we want to keep the `h1` as `Your teacher training application` ie the on-page heading? If so, you can add a `heading:` that is different from `title:`

But if you are happy with the h1 not having `Your`, that's fine with me!